src/Delivery/Cache/FilesystemCache.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @copyright 2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Delivery\Cache;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+class FilesystemCache implements CacheInterface
+{
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    /**
+     * @var string
+     */
+    private $cacheDir;
+
+    /**
+     * FilesystemCache constructor.
+     *
+     * @param  string $cacheDir
+     * @param  string $spaceId
+     */
+    public function __construct($cacheDir, $spaceId)
+    {
+        $this->fs = new Filesystem;
+        $this->cacheDir = $cacheDir . '/' . $spaceId;
```

Comment:
excuse my PHP, isn't there a function to do a join instead of specifying `'/'` manually